src/main.rs

```diff
@@ -100,17 +100,28 @@ fn daemonize() -> Result<()> {
 
 fn run() -> Result<()> {
 
-	#[cfg(unix)]
-	daemonize()?;
-
 	// Parse CLI options
 	let args: Vec<String> = std::env::args().collect();
 	let mut options = Options::new();
 	options.optopt("c", "config", "set the configuration file", "FILE");
 	options.optopt("d", "database", "set the path to index database", "FILE");
 	options.optopt("w", "web", "set the path to web client files", "DIRECTORY");
+	options.optflag("f", "foreground", "run polaris in the foreground instead of daemonizing");
```

Comment:
Polaris never runs as a daemon on Windows, can you add the #[cfg(unix)] conditional compilation flag here?